plugin.service.gto/addon.xml

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.service.gto" name="German Telecast Offers" version="2.0.4" provider-name="Birger Jesch">
+    <requires>
+        <import addon="xbmc.python" version="2.20.0"/>
+    </requires>
+    <extension point="xbmc.python.script" library="default.py" />
+    <extension point="xbmc.service" library="starter.py" start="startup">
```

Comment:
ok, thx for explaining. Fine for me then.

I didnt even know that it can be used as a plugin if that extension point is not defined, so I also learned something. :)